### ISSUE TITLE
Feature Request: configurable image viewer

### ISSUE THREAD

**Issue by Glog78** [wustho/epr#55]
It would be totally amazing if we could have a configurable image viewer. I suggest that even being fully aware that i could already configure the default image viewer using xdg standard. The reason for this is mostly i don't want to change the default image viewer for other applications (it makes sense to have a more feature rich solution there). There is a second advantage without writing an additional bash script we can setup parameter for the image viewer that way.

**Comment by jneidel** [comment]
Yeah, this sounds convenient. I don't use xdg and always just edit the order in the source file, because not having the first entry installed just does not open the image, despite having my image viewer further down the list.
